saleor/checkout/migrations/0040_auto_20210920_0934.py

```diff
@@ -0,0 +1,86 @@
+# Generated by Django 3.2.7 on 2021-09-20 09:34
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+def set_default_checkout_line_currency(apps, schema_editor):
+    CheckoutLine = apps.get_model("checkout", "CheckoutLine")
+    for checkout_line in CheckoutLine.objects.all().iterator():
```

Comment:
IMO Have tried to use F expression to handle this case? or something different. This can cause a huge down time as iterating over all lines could take some time